projects/solar/src/solar_config.c

```diff
@@ -136,6 +137,12 @@ const GpioAddress *config_get_drv120_relay_pin(void) {
   return &s_drv120_relay_pin;
 }
 
+// To enable status interrupt for drv120, add pin below, and return its address here
+const GpioAddress *config_get_drv120_status_pin(void) {
+  // return &s_drv120_status_pin;
+  return NULL;
+}
```

Comment:
If you want to make it easy to disable the status pin, I'd suggest a `#define` at the top like `#define ENABLE_DRV120_FAULT_HANDLING true`. Then here you can do:
```c
return ENABLE_DRV120_FAULT_HANDLING ? &s_drv120_status_pin : NULL;
```